ros2action/ros2action/verb/send_goal.py

```diff
@@ -117,7 +123,9 @@ def send_goal(action_name, action_type, goal_values, feedback_callback):
             return 'Failed to populate message fields: {!r}'.format(ex)
 
         print('Waiting for an action server to become available...')
-        action_client.wait_for_server()
+        if not action_client.wait_for_server(timeout_sec=timeout):
+            print(f'Action server is not available after waiting {timeout} seconds.')
+            return
```

Comment:
```suggestion
            return f'Action server is not available after waiting {timeout} seconds.'
```
This value will eventually be passed to sys.exit(). If it's a string, it will be printed to stderr and the exit code will be set to 1.  If it's none, exit code will be set to 0.  
For timeout, I think it is better to return 1.   
What do you think?

Reply:
that will fail the test since we cannot catch the expected output, besides it the command becomes the failure.

Reply:
Okay. While timeout occurs,  the exist code of command is 0.